cmd/openshift-install/testdata/agent/image/configurations/sno.txt

```diff
@@ -13,10 +13,10 @@ apiVersion: v1
 baseDomain: test.metalkube.org
 controlPlane: 
   name: master
-  replicas: 1
+  replicas: 3
```

Comment:
Because this is named sno.txt, it is odd the masters replicas is changed to 3. I think we should leave this one unchanged and create another integration test for none platform and either compact or HA cluster configuration.